package.json

```diff
@@ -177,11 +180,24 @@
     ],
     "env": {
       "browser": true,
+      "node": true,
       "jest": true,
       "es6": true
     },
     "rules": {
-      "prettier/prettier": "error",
+      "import/no-unresolved": 0,
+      "import/no-extraneous-dependencies": [
```

Comment:
Where was this erroring?

Reply:
These are used to avoid import errors when importing directly from 'src/containers': 

<img width="1054" alt="Screenshot 2022-12-19 at 11 04 15 AM" src="https://user-images.githubusercontent.com/6717198/208500557-bed51718-3c17-43bf-b147-8175590daf25.png">

Reply:
Did you try using this? https://www.npmjs.com/package/eslint-import-resolver-typescript

Reply:
We use relative URLs, which doesn't have this issue, so for consistency it's probably better to do that.

Does `src/containers/...` work?

Reply:
no src/containers doesn't work.

Reply:
I disagree with this line of reasoning - i think this kind of importing is better since it allows us to move files around more easily but we need to get this merged so I'm going to make this change.

Reply:
My biggest worry is that since eslint doesn't support this, IDEs won't, and you can't use the Ctrl-Click feature to go to files. I find this _incredibly_ useful.

Reply:
@Amindv1 lets remove the absolute paths for now.